CONTRIBUTING.rst

```diff
@@ -83,12 +83,16 @@ Releases are created using `manual GitHub workflows
 <https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow>`_.
 As a maintainer, follow these steps:
 
-1. Create a new branch
-2. Bump the version defined in ``skops/__init__.py``
-3. Git grep for any TODO's that need fixing before the release (e.g.
+1. Check and update the ``CHANGES.md``
+2. For a major release, create a new branch with the name "0.version.X", e.g.
```

Comment:
Technically, according to semantic versioning, this would be a "minor" release. I think many people use "major" the way you do here, I'm not sure if we want to strictly follow semver.

Reply:
Yeah we don't, the whole ecosystem doesn't so I don't think it's a big deal here.